docs/sequenceDiagrams/deployBlob.md

```diff
@@ -0,0 +1,17 @@
+```mermaid
+sequenceDiagram
+  participant s as Serverless CLI
+  participant r as Resource Group 
+  participant f as Function App
+  participant b as Blob Storage
+
+  note right of s: `sls deploy`
+  s ->> r: Create resource group
+  s ->> r: Deploy ARM template
+  r ->> f: Included in ARM template
+  r ->> b: Included in ARM template
+  note right of s: Zip code
+  s ->> b: Deploy zip code with name {appName}-v{version}.zip
```

Comment:
As per current design, this should include timestamp/commit SHA, instead of version.  It can stay if we want -- just need a more unique value as well.